src/components/BnListbox/BnListbox.spec.ts

```diff
@@ -180,4 +180,27 @@ describe('BnListbox', () => {
       expect(button.classes().includes('bn-listbox__button--error')).toBe(true);
     });
   });
+
+  describe('hidden inputs', () => {
```

Comment:
Maybe you could add some context to each it in this test.  Like with one option value or with multiples option values